waltz-tools/src/main/java/com/wepay/waltz/tools/storage/StorageCli.java

```diff
@@ -272,7 +273,7 @@ protected void configureOptions(Options options) {
                     .build();
             Option partitionOption = Option.builder("p")
                     .longOpt("partition")
-                    .desc("Specify the partition id to be added to the storage node")
+                    .desc("Specify the preferred partition (or multiple partitions as comma-separated int ranges such as 0-6,7,10-16) to be added to the given server")
```

Comment:
May be better to remove the `preferred` keyword as it might be confused with the `preferred partitions` concept that we have on the WaltzServer. 
Other places in this files needs to be similarly updated.